/**:
  ros__parameters:
    feedback_hz: 50
    status_hz: 20
    debug_hz: 20
    can_bus: 'can0'
    joint_names: ['rear_left_wheel_joint']
    joint_can_ids: [101]
    joint_directions: [1]
